src/cloud/components/organisms/settings/SettingsComponent.tsx

```diff
@@ -43,6 +25,10 @@ import IntegrationsTab from './IntegrationsTab'
 import PreferencesTab from './PreferencesTab'
 import ApiTab from './ApiTab'
 import { PageStoreWithTeam } from '../../../interfaces/pageStore'
+import SettingsLayout from '../../../../shared/components/organisms/Settings/molecles/SettingsLayout'
```

Comment:
Two things: 

`molecles` => `molecules`


Secondly: 

In this case the `SettingsLayout` should be the default export from `shared/components/organisms/Settings` instead in my opinion.